packages/wallets/src/features/cfd/screens/Onfido/Onfido.tsx

```diff
@@ -1,15 +1,40 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
 import { useOnfido } from '@deriv/api';
+import { useFlow } from '../../../../components/FlowProvider';
+import { WalletsActionScreen } from '../../../../components/WalletsActionScreen';
```

Comment:
Just a question but couldn't these imports be combined under `../../../../components`?